README.md

```diff
@@ -8,75 +8,92 @@
   <a href="https://www.npmjs.com/package/@algolia/netlify-plugin-crawler"><img src="https://img.shields.io/npm/v/algolia/netlify-plugin-crawler" alt="Version"></a>
 </p>
 
-<h2 align="center">Algolia Netlify integration</h2>
+<h2 align="center">Algolia Netlify plugin</h2>
 
-Index your website automatically to Algolia when you deploy your project to Netlify, using our dedicated Crawler.
+Automatically index your website to Algolia when deploying your project to Netlify with the Algolia Crawler.
 
 - [What is Algolia?](https://www.algolia.com/doc/guides/getting-started/what-is-algolia/)
 - [What is Algolia's Crawler?](https://www.algolia.com/doc/tools/crawler/getting-started/overview/)
 
-## Getting Started
+## Table of contents<!-- omit in toc -->
 
-In this getting started we will implement an instant search using Netlify, Algolia, and this plugin.  
-To start right away **you only need a Netlify site**.
+- [Getting started](#getting-started)
+  - [Link your site to Algolia](#link-your-site-to-algolia)
+  - [Indexing](#indexing)
+  - [Install the front-end bundle](#install-the-front-end-bundle)
+  - [Going further](#going-further)
+- [Uninstall the plugin](#uninstall-the-plugin)
+- [Limitations, known issues, and FAQ](#limitations-known-issues-and-faq)
+  - [My website requires JavaScript to render](#my-website-requires-javascript-to-render)
+  - [I want to extract more information from my pages](#i-want-to-extract-more-information-from-my-pages)
+  - [Can I build my own UI?](#can-i-build-my-own-ui)
+  - [Can I receive a notification when my crawl is done?](#can-i-receive-a-notification-when-my-crawl-is-done)
+- [Troubleshooting](#troubleshooting)
+
+## Getting started
+
+Here's how you can implement an instant search on your site using Netlify, Algolia, and this plugin. **All you need is an existing Netlify site.**
 
 ### Link your site to Algolia
 
-**Go to <https://crawler.algolia.com/admin/netlify>**
+First, go to the [Algolia Crawler for Netlify](https://crawler.algolia.com/admin/netlify) and click **Sign in to Algolia with Netlify**. We automatically create a new Algolia account if you don't have one.
```

Comment:
Are users redirected to the dashboard to create their account, or do we create an account for them programmatically?

If the latter, how do they retrieve their dashboard credentials?

Reply:
The account are automatically created unless the account exists and they are not logged, in that case they will be asked to connect before; 
then no matter what they will be redirected to the crawler